GoMeet/src/main/webapp/js/createMeetingStep2.js

```diff
@@ -0,0 +1,41 @@
+const MAX_HOURS = 23; // maximum number of hours in hour input of meeting duration
+const MAX_MINS = 59; // max number of minutes in minute input of meeting duration
+const ZERO_DURATION_ALERT = 'Duration should be at least 1 minute';
+const INVALID_DURATION_ALERT = 'Please input a valid duration: 0-23 hours 0-59 minutes';
+
+/**
+ * Validate that the hours and minutes entered for meeting duration are 
+ * valid.
+ * The duration should at least be 1 minute, and hours is a value 0-23
+ * and minutes 0-59. Alerts the user to input valid value(s)
+ * @returns true if duration is valid, false otherwise.
+ */
+function validateDuration() {
+  let hours = document.getElementById('duration-hours').value;
+  let mins = document.getElementById('duration-mins').value;
+  if (hours === '' || mins === '') {
+    alert(FILL_ALL_FIELDS_ALERT);
```

Comment:
@suprsrsly Should I also return status codes here as well and alert at a higher layer?

Reply:
yeah, I think so. Displaying alerts inside a validateFoo() function doesn't seem right because it might complicate testing.